src/injected/styles/injected.scss

```diff
@@ -131,13 +131,14 @@
         }
     }
 
-    .file-issue-button,
-    .copy-issue-details-button,
-    .insights-dialog-button-inspect {
+    .insights-dialog-main-override .file-issue-button,
```

Comment:
The scoping here was required to get the fill override to work. It's specified everywhere else in this file and should have been included in #4253